api/comments_api.py

```diff
@@ -96,6 +96,12 @@ def do_post(self, feature_id, field_id=None):
           new_approval_state=new_state)
       comment.put()
 
+      # Schema migration double-write.
+      comment_activity = review_models.Activity(
+        id=comment.key.integer_id(), feature_id=feature_id, gate_id=field_id,
```

Comment:
Nice.  I thought it would require creating a `Key` object rather than just setting the `id` directly.

Reply:
Yeah, I did too due to needing Key objects for filtering, but it seems it's easier than expected. 😊